modules/backend/widgets/Filter.php

```diff
@@ -264,6 +264,14 @@ public function onFilterUpdate()
                 $value = post('options.value.' . $scope->scopeName) ?: null;
                 $this->setScopeValue($scope, $value);
                 break;
+
+            case 'clear':
+                foreach ($this->getScopes() as $scope) {
+                    $this->setScopeValue($scope, null);
+                }
+
+                return ['.control-filter' => $this->render()];
+                break;
```

Comment:
@robinbonnes A `break` directive is not needed if there's a `return` above it in the same scope.
```suggestion
```